### DISCUSSION TITLE
Granular Grid view

### DISCUSSION THREAD

**Discussion by bernhardh** [orchidsoftware/platform#1740] [Q&A]
Is there a way to use `Layout::columns` (or any other component) to make a2 columns layout, where one is 2/3 and the other is 1/3 (or 3/4 vs 1/4) wide?

The idea behind that is to create an edit screen where I have the form in the right column and aditional information and even maybe actions in a sidebar on the right. Something like this:

![image](https://user-images.githubusercontent.com/642292/119223187-4e1b4d80-baf8-11eb-80a4-4f53ac84ef80.png)

**Comment by AnhTuanDev** [comment]
Don't know if you've read this yet.
Hope you mean it.

`https://orchid.software/en/docs/custom-template/#wrapper`
